src/main/resources/templates/tutorials.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>National Cybersecurity Laboratories - Experiment</title>
+        <object th:include="fragments/header :: header" th:remove="tag" />
+    </head>
+    <body class="mybg loginbackground">
+        <div th:replace="fragments/login_navigation :: loginnav">&nbsp;</div>
+        <div class="container">
+            <p>&nbsp;</p>
+            <p>&nbsp;</p>
+            <div class="heading h15 text-center">
+                <p>Tutorials</p>
+            </div>
+            <div class="row dashpanel">
+                <h2>1) How to Create an Experiment </h2>
+                    <div class="heading col-sm-12 ">
+
+                        <div class="col-sm-6">
+                            <img preload="true" minimal-lightbox="images/createexperiment2.png" src="images/createexperiment2.png" width="500" height="315"/></div>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add an "alt" attribute to this image. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)